uint8 home
uint8 run
uint8 rest

float64[6] joint_angles

